scan/lib/assets/ScanfileTemplate

```diff
@@ -1,5 +1,5 @@
 # For more information about this configuration visit
-# https://github.com/fastlane/scan#scanfile
+# https://github.com/fastlane/fastlane/tree/master/scan
```

Comment:
Nice catch, thanks for the PR. The link should be https://github.com/fastlane/fastlane/tree/master/scan#scanfile however 👍